dags/tasks/amazon/s3.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import os
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+from pymarc import MARCReader
+from urllib.parse import urlparse
+
+
+def get_from_s3(**kwargs) -> list:
+    urls = kwargs.get("urls")
+    s3_hook = S3Hook(aws_conn_id="aws_lambda_connection")
+    downloaded_files = []
+
+    for url in urls:
```

Comment:
We should only get a single BF Instance URL from the SQS message and not a list of URLs.